apps/site/pages/docs/api-extensions.mdx

```diff
@@ -5,36 +5,44 @@ import { ExtendableQueryTable } from './../../components/ExtendableQueryTable'
 
 # API extensions
 
-In this guide you will learn how to extend the FastStore API schema.
+In this guide, you will learn how to use the FastStore API Extension.
 
 </header>
 
+1. Define which new fields will be exposed by the FastStore API Extension.
+2. Determine the GraphQL fragments where these fields will be used.
+3. How to consume the new fields using [New Sections](/docs/building-sections/creating-a-new-section) or [Custom Overridable Components](/docs/building-sections/overriding-components-and-props#overriding-a-native-component).
+
 Although the [FastStore API](https://v1.faststore.dev/reference/api/faststore-api) provides a [GraphQL schema for ecommerce](https://v1.faststore.dev/reference/api/queries) that suits a general use case, some stores may need to access other, more specific, information.
 
-You will not be able to add other APIs endpoints to your storefront since this may [impact site performance](https://v1.faststore.dev/how-to-guides/faststore-api/fetching-api-data#best-practices-for-fetching-data). If you need to fetch more data, it is possible to extend the FastStore API schema, adding new data to the existing [queries](https://v1.faststore.dev/reference/api/queries).
+It's not recommended to add other API endpoints to your storefront directly, since this may [impact site performance](https://v1.faststore.dev/how-to-guides/faststore-api/fetching-api-data#best-practices-for-fetching-data). 
```

Comment:
We can use `Callout` component for this info.

Reply:
nice! done [here](https://github.com/vtex/faststore/pull/1998/commits/bc9e9af3f639d74cbe86a543bea36335ab9ad201).